examples/cirrus-ci-minimal.yml

```
@@ -20,6 +20,20 @@ linux_x86_task:
     - apt install -y python3-venv python-is-python3
   <<: *BUILD_AND_STORE_WHEELS
 
+linux_arm64_task:
+  name: Build Linux arm64 wheels.
```

Python calls this architecture `aarch64`, as does cibuildwheel, so it's probably best to stick to the Python convention here.

```suggestion
linux_aarch64_task:
  name: Build Linux aarch64 wheels.
```